Classes/Extensions/DDMultiFormatter.m

```diff
@@ -75,27 +73,19 @@ - (void)dealloc {
 #pragma mark Processing
 
 - (NSString *)formatLogMessage:(DDLogMessage *)logMessage {
-    __block NSString *line = logMessage->_message;
+    DDLogMessage *message = [logMessage copy];
 
     dispatch_sync(_queue, ^{
         for (id<DDLogFormatter> formatter in _formatters) {
-            DDLogMessage *message = [self logMessageForLine:line originalMessage:logMessage];
-            line = [formatter formatLogMessage:message];
+            message->message = [formatter formatLogMessage:message];
```

Comment:
We don't want to "crush" the original message with a formatted one. For example we may want to have different formatters for each logger with longer/shorter log outputs.